.changes/unreleased/Fixes-20240607-102708.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Fixes
+body: return to previous naming convention to return to quoting policy
```

Comment:
I like to think of reading these as an end user. From that perspective, I don't think it's clear what the change is without a ton more context. I would re-phrase to emphasize _what_ users get versus _how_ we solved it for them.